k8s/aat/common/financial-remedy/finrem-cos.yaml

```diff
@@ -26,6 +26,8 @@ spec:
         FEATURE_HWF_SUCCESSFUL_NOTIFICATION_LETTER: true
         FEATURE_ASSIGNED_TO_JUDGE_NOTIFICATION_LETTER: true
         FEATURE_CONSENT_ORDER_NOT_APPROVED_APPLICANT_DOCUMENT_GENERATION: true
+        BSP_SERVICES_ALLOWED_TO_UPDATE: "bulk_scan_orchestrator"
+        BSP_SERVICES_ALLOWED_TO_VALIDATE: "bulk_scan_processor"
```

Comment:
service names are supposed to be in quotes? Is there something in the code that will strit the quotes?